### DISCUSSION TITLE
Dual Frequency Channels 2.4 Ghz and 5.8 Ghz on Mavlink

### DISCUSSION THREAD

**Discussion by koklimabc** [svpcom/wfb-ng#327] [Questions]
I'm decided to occupy UART RX6 and TX6 of FC "MATEK H743 Slim v3" for FM30 receiver to work as mavlink birectional communication.

My WFB-NG OPENIPC FPV Camera requires RX and TX must connect TX and RX UART of Flight Controller under mavlink protocol, then forwards video signal through RTL8812AU WIFI module

I do knowed about 2.4 Ghz channel and 5.8 Ghz channel are totally differently and cannot be used in together but both using UDP 14550 and 14551 for tranmission.

Does open one birectional Mavlink on UART RX6 and TX6 could be used together for both 5.8 Ghz and 2.4 Ghz frequency channel as long as WFB-NG promised not problems on https://github.com/svpcom/wfb-ng/discussions/304

or maybe else I could possible open another softserial as smartport on UART for bidirectional communication under FC MATEK H743 slim or any other better ways?

**Comment by koklimabc** [comment]
![discord](https://github.com/svpcom/wfb-ng/assets/2577622/8d8e3958-489e-479a-a5e9-a703b9ee45aa)
I'd learned that WFB-NG is a single telemetry link send it parallel, in case FC UART RX6 and TX6 used for mavlink 2.4Ghz receiver, I could add solder point from same FC side's TX6 into RX of SCC338Q module and just leave SSC338Q side's TX without touch with FC side's RX6?

I'm using ardupilot.

**Comment by svpcom** [comment]
Do you mean openipc implementation of mavlink forwarder? This is not a part of official wfb-ng.
All features in https://github.com/svpcom/wfb-ng/discussions/304 are related to full-featured wfb-ng (which openipc doesn't use due to memory and flash limits).

Yes, wfb-ng can use 2.4 + 5.8 together but this will require two cards (and usb hub), because single card can use only one frequency channel simultaneously.

**Reply by koklimabc** [reply]
i see, It seems like I need to open another FC side's softserial for two way communication under 5.8 GHz channel.
